packages/@react-aria/actiongroup/src/useActionGroupItem.ts

```diff
@@ -44,10 +44,11 @@ export function useActionGroupItem<T>(props: AriaActionGroupItemProps, state: Li
 
   let isFocused = props.key === state.selectionManager.focusedKey;
   let lastRender = useRef({isFocused, state});
-  lastRender.current = {isFocused, state};
+  useEffect(() => {
```

Comment:
this translation to useEffect is safe since we only access this raf in later useEffects, so this will always run between the last render and the next useEffect